packages/arb-ts/scripts/instantiate_bridge.ts

```diff
@@ -7,24 +7,29 @@ dotenv.config()
 
 const pk = process.env['DEVNET_PRIVKEY'] as string
 const mnemonic = process.env['DEV_MNEMONIC'] as string
+const verbose = process.env['VERBOSE'] as string
+
 const defaultNetworkId = 4
```

Comment:
should we allow an env variable to override the default network id?

Reply:
so that's what the --networkID flag does...

Reply:
kind nice having this be a command line flag (at least in my experience